components/partials/portfolio/open-positions.vue

```diff
@@ -31,9 +48,89 @@ export default Vue.extend({
     }
   },
 
+  computed: {
+    isUserWalletConnected(): boolean {
+      return this.$accessor.wallet.isUserWalletConnected
+    },
+
+    markets(): UiDerivativeMarket[] {
+      return this.$accessor.derivatives.markets
+    },
+
+    positions(): UiPosition[] {
+      return this.$accessor.portfolio.subaccountPositions
+    }
+  },
+
   methods: {
     onSelect(component: string) {
       this.component = component
+    },
+
+    getMarket(marketId: string): UiDerivativeMarket | undefined {
+      const { markets } = this
+
+      return markets.find((m) => m.marketId === marketId)
+    },
+
+    getPrice(position: UiPosition): BigNumberInBase {
```

Comment:
I don't like this approach. What we should do instead is get all of the markets from the `derivatives.ts` store module, filter them down based on the positions we are trying to close and have a helper function (same as we have for `calculateMargin`) that returns the `feeAdjustedBankruptcyPrice` given a position and a market.